lib/mappers/bhl_mods.py

```diff
@@ -42,17 +41,14 @@ def map_description(self):
         if exists(self.provider_data, prop):
             description = []
             for s in iterify(getprop(self.provider_data, prop)):
-                try:
-                    if s.get("type") == "content":
-                        description.append(s.get("#text"))
-                except:
-                    logger.error("Error getting note/type from record %s" %
-                                 self.provider_data["_id"])
+                if "type" in s and s.get("type") == "content":
+                    description.append(s.get("#text"))
 
-            self.update_source_resource({"description": description})
+            if description:
+                self.update_source_resource({"description": description})
 
-    # OAI MODS Mapper is blowing up on this method because of arg c
-    # ount issues, even though we handle it below
+    # OAI MODS Mapper is blowing up on this method because of arg
+    # count issues,
```

Comment:
Also, in our MAPv4 schema, `dc:format`'s obligation is 0-n, and so is `dcterms:type`'s, although it is "strongly recommended."